hotsos/core/plugins/kernel/common.py

```diff
@@ -1,10 +1,46 @@
 import os
 import re
 from functools import cached_property
+from datetime import datetime
 
+from hotsos.core.host_helpers.cli import CLIHelper
 from hotsos.core import host_helpers, plugintools
 from hotsos.core.config import HotSOSConfig
+from hotsos.core.log import log
 from hotsos.core.plugins.kernel.config import KernelConfig
+from hotsos.core.plugins.system import SystemBase
+
+
+# NOTE(dosaboy): when updating this, refer to the kernel supported versions
+# page: https://ubuntu.com/about/release-cycle#ubuntu-kernel-release-cycle
+KERNEL_EOL_INFO = {
+    '5.19': 
```

Comment:
The page includes the kernel EOL info for 6.5, but 5.19 is now absent (IDK if it was ever present).

Also, I found the [distro-info-data](https://salsa.debian.org/debian/distro-info-data/) Debian package which includes [this](https://salsa.debian.org/debian/distro-info-data/-/blob/main/ubuntu.csv?ref_type=heads) CSV file with all the ubuntu release lifecycle information. Still, it does not include the supported kernels for each release --  might be useful for checking the release itself's EOL status.